src/DataStructures/Tensor/EagerMath/CrossProduct.hpp

```diff
@@ -0,0 +1,178 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines functions euclidean cross_product and cross_product with a metric
```

Comment:
I think this reads a little better:  "Defines functions `cross_product` for flat and curved-space cross products."